webpack.config-frontend.js

```diff
@@ -73,6 +73,7 @@ module.exports = {
     login_password_recover: './frontend/www/js/omegaup/login/recover.ts',
     login_password_reset: './frontend/www/js/omegaup/login/reset.ts',
     logout: './frontend/www/js/omegaup/login/logout.ts',
+    problem_collections: './frontend/www/js/omegaup/problem/collection.ts',
```

Comment:
nit:

```suggestion
    problem_collection: './frontend/www/js/omegaup/problem/collection.ts',
```

por consistencia con el nombre del archivo